### PR TITLE
Implement content info metrics (word, characters) in Gutenberg.

### PR DESCRIPTION
**Pull request by SergioEstevao**
Fixes: https://github.com/wordpress-mobile/gutenberg-mobile/issues/506

GB-Mobile PR: https://github.com/wordpress-mobile/gutenberg-mobile/pull/2380

To test:
 - Start the app
 - Open a post using the Block editor
 - Tap on the ... button on the top right
 - Check if the post metrics are displayed on the top of action sheet
 

PR submission checklist:

- [x] I have considered adding unit tests where possible.
- [x] I have considered adding accessibility improvements for my changes.
- [x] I have considered if this change warrants user-facing release notes and have added them to `RELEASE-NOTES.txt` if necessary.

### CODE REVIEW

**Comment by SergioEstevao** [comment]
> There seems to be a small but important detail I commented on code. The rest looks shinny 

I don't see the comment...

File: WordPress/Classes/ViewRelated/Gutenberg/GutenbergViewController+MoreActions.swift [hunk 1]
```diff
@@ -12,13 +12,12 @@ extension GutenbergViewController {
     func displayMoreSheet() {
         let alert = UIAlertController(title: nil, message: nil, preferredStyle: .actionSheet)
 
-        //TODO: Comment in when bridge is ready
-        /*if mode == .richText {
+        if mode == .richText, let contentInfo = contentInfo {
             // NB : This is a candidate for plurality via .stringsdict, but is limited by https://github.com/wordpress-mobile/WordPress-iOS/issues/6327
-            let textCounterTitle = String(format: NSLocalizedString("%li words, %li characters", comment: "Displays the number of words and characters in text"), richTextView.wordCount, richTextView.characterCount)
+            let textCounterTitle = String(format: NSLocalizedString("%li block(s), %li word(s), %li character(s)", comment: "Displays the number of blocks, words and characters in text"), contentInfo.wordCount, contentInfo.characterCount)
```
Comment (etoledom) [inline WordPress/Classes/ViewRelated/Gutenberg/GutenbergViewController+MoreActions.swift:17]: The variable for `%li block(s)` seems to be missing here. It gives wrong numbers to the display.

**Comment by etoledom** [comment]
> I don't see the comment...

@SergioEstevao - Sorry, I don't know what happened! I wrote it again 👍

File: WordPress/Classes/ViewRelated/Gutenberg/GutenbergViewController+MoreActions.swift (hunk 1 above)
Comment (SergioEstevao) [reply WordPress/Classes/ViewRelated/Gutenberg/GutenbergViewController+MoreActions.swift:17]: Good catch!

Comment (SergioEstevao) [reply WordPress/Classes/ViewRelated/Gutenberg/GutenbergViewController+MoreActions.swift]: I fixed it, do you mind give it another go @etoledom ?

**Comment by jkmassel** [comment]
Howdy friends – we're freezing `15.1` today, so I'm bumping this PR forward to `15.2`. If it needs to land in the `15.1` series, please merge it to `release/15.1` and DM me – I'll be happy to create a new beta release!

**Pull request merged**